src/west/util.py

```diff
@@ -53,6 +53,22 @@ class WestNotFound(RuntimeError):
     '''Neither the current directory nor any parent has a west workspace.'''
 
 
+def list_to_str(in_list: list, sep: str = os.pathsep) -> str:
+    """Convert a list of items to a single string joined by the given separator."""
+    return sep.join([str(x) for x in in_list])
```

Comment:
I think this particular one-liner saves little typing and obscures a simple thing.